src/shared/components/table/styles/table.module.scss

```diff
@@ -1,3 +1,11 @@
+$table__header-text: #393b42;
+$table__header-background: #fff;
+$table__even: #fff;
+$table__odd: #e4e8eb;
+$table__header-text: #393b42;
+$table_border: #c2c4c4;
+$table_hover: #f1f1f1;
```

Comment:
Do we really want to keep using SASS variables on new or touched code?

If the point is to reuse the same colours throughout this file in different places these could be defined as CSS custom properties on `:root` or on a containing class (for example here `.container`) and used later on.
If we want to define them as internal variables with defaults and then allow them to be overridden that would still work as long as whoever overrides them does so on a lower component within the DOM tree (so it'd be better to define on `:root` to avoid issues)

Also, I see that `$table__header-text` is defined twice

Reply:
I did this to reuse `$table__header-text` in the community curation style sheet for consistency. Can we do this without sass variables?

Reply:
Done as suggested